make/common/postgresql/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM library/photon:1.0
+
+ENV PGDATA /var/lib/postgresql/data
+
+RUN touch /etc/localtime.bak \
+    && tdnf install -y sed shadow gzip postgresql\
+    && groupadd -r postgres --gid=999 \
+    && useradd -r -g postgres --uid=999 postgres \
+    && mkdir -p /run/postgresql \
+    && chown -R postgres:postgres /run/postgresql \
+    && chmod 2777 /run/postgresql \
+    && mkdir -p "$PGDATA" && chown -R postgres:postgres "$PGDATA" && chmod 777 "$PGDATA" \
+    && sed -i "s|#listen_addresses = 'localhost'.*|listen_addresses = '*'|g" /usr/share/postgresql/postgresql.conf.sample \
+    && sed -i "s|#unix_socket_directories = '/tmp'.*|unix_socket_directories = '/run/postgresql'|g" /usr/share/postgresql/postgresql.conf.sample \
+    && touch /usr/share/locale/locale.alias \
+    && locale-gen.sh en_US.UTF-8
+
```

Comment:
I recall there's a xxxx-initdb.d directory and when the sql file is put there it will be executed automatically?
Could you investigate that, it's fine if it's covered in future work item.